config/v1/types_infrastructure.go

```diff
@@ -118,6 +126,21 @@ type AWSPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// AzurePlatformStatus holds the current status of the Azure infrastructure provider.
+type AzurePlatformStatus struct {
+	// resourceGroupName is the Resource Group for new Azure resources created for the cluster.
+	ResourceGroupName string `json:"resourceGroupName"`
```

Comment:
Why not `resourceGroup` instead of `resourceGroupName`?  That would be congruent to `region`.  Is there a `resourceGroupUID` or something?

Reply:
Yeah, the region is always the name, but resource group can have the complete id, `/subscriptions/<sid>/resourcesgroups/<rresource group name>`
with resource group name, you cannot specify the subscription ID, which we do not want.